Project.toml

```diff
@@ -8,10 +8,10 @@ Colors = "0.12, 0.11, 0.10"
 DataFrames = "1"
 DoubleFloats = "1"
 FileIO = "1"
-HTTP = "1.10"
+HTTP = "1"
 JSON = "0.21"
 Measurements = "2"
-MeshIO = "0.4.10"
+MeshIO = ">= 0.4.10"
```

Comment:
`>=` won't work when releasing it

Reply:
Why not? This is standard Pkg.jl syntax: https://pkgdocs.julialang.org/v1/compatibility/#Inequality-specifiers

Reply:
But automerging failed for the last release. It might be fine for Pkg.jl but not for the Julia registry [https://juliaregistries.github.io/RegistryCI.jl/stable/guidelines/](https://juliaregistries.github.io/RegistryCI.jl/stable/guidelines/)